### DISCUSSION TITLE
涨价去库存5周年：中国房地产到底发生了什么？

### DISCUSSION THREAD

**Discussion by zsmhub** [WeNeedHome/SummaryOfLoanSuspension#926] [General]
[点击查看原文](https://posts.careerengine.us/p/622dd49888088b2b49581af5)

可以说当年轰轰烈烈的“涨价去库存”仅仅造福了少数地产商，拆迁户、炒房者和持有多套房的城市土著也跟着喝汤，实际上根本没太起到去企业杠杆的作用。

可与此同时，居民杠杆已然直接拉满，消费潜力榨干，从更长远来看，代表未来劳动力供给的生育率也断崖下滑。“涨价去库存”的苦果，可能需要未来数十年来偿还。

本文的摘要是“涨价去库存还会发生吗”，读完全文的您，觉得这个问题还有回答的必要么？

**Comment by jingzz1** [comment]
不得不说当年提出棚改货币化的那位强人，的确创造了一个世界经济史上的神话，虽然把企业杠杆转移到居民身上后，企业又进一步增加杠杆，导致杠杆率进一步拉高，但确实也挽救了当时的中国经济，如果不是疫情，房地产经济估计还能撑多几年。
